### ISSUE TITLE
HintParseException after upgrading 1.4.0 -> 1.4.1

### ISSUE THREAD

**Issue by juergenzimmermann** [jeremylong/dependency-check-gradle#14]
After upgrading from 1.4.0 to 1.4.1 I'm getting a `HintParseException` regarding the "false positives" file. The error message:
```
...
Analysis Starting
Unable to parse hint rule xml file 'C:\...\kunden\src\test\resources\owasp.xml'
org.owasp.dependencycheck.xml.hints.HintParseException: org.xml.sax.SAXException Line=2, Column=99: cvc-elt.1: Deklaration des Elements 'suppressions' kann nicht gefunden werden.
Exception occurred initializing Hint Analyzer.
```
Translation of the German Phrase above:
```
The deklaration of the element 'suppressions' cannot be found.
```
The first 2 lines of owasp.xml:
```
<?xml version="1.0"?>
<suppressions xmlns="https://jeremylong.github.io/DependencyCheck/dependency-suppression.1.1.xsd">
```
See https://jeremylong.github.io/DependencyCheck/general/suppression.html

**Comment by juergenzimmermann** [comment]
Translation of the German Phrase above:
```
The deklaration of the element 'suppressions' cannot be found.
```

**Comment by juergenzimmermann** [comment]
I get the same error when I try
```
<suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
              xmlns="https://www.owasp.org/index.php/OWASP_Dependency_Check_Suppression"
              xsi:schemaLocation="https://www.owasp.org/index.php/OWASP_Dependency_Check_Suppression suppression.xsd">
```
as in https://github.com/jeremylong/DependencyCheck/blob/master/dependency-check-core/src/test/resources/suppressions.xml

**Comment by jeremylong** [comment]
Apparently, I caused this due to a copy/paste error when updating the Hunt Analyzer. If you specified a suppression file the hint analyzer tries to parse it (as well as the suppression analyzers).  The code has been updated and a new version will be published shortly as this will cause several folks issues.

**Comment by jeremylong** [comment]
1.4.2 has been published.

**Comment by juergenzimmermann** [comment]
@jeremylong 1.4.2 works fine.
